api.planx.uk/modules/webhooks/controller.ts

```diff
@@ -188,3 +191,33 @@ export const isCleanJSONBController: IsCleanJSONBController = async (
     );
   }
 };
+
+export const updateTemplatedFlowEditsController: UpdateTemplatedFlowEditsController =
+  async (_req, res, next) => {
+    const { flowId, templatedFrom, data } = res.locals.parsedReq.body.payload;
+
+    try {
+      if (isNull(templatedFrom)) {
+        return res.status(200).send({
+          message: `Not a templated flow, skipping updates (${flowId})`,
+        });
+      }
```

Comment:
Ideally, we'd be able to define a trigger that listens for updates on `flows.data` column only **`where flows.templated_from is not null`** - but Hasura doesn't support the second condition. 

So this means we'll have an event for _every_ flow update independent of whether or not it's a template.

At best, we can ensure that `templatedFrom` is directly passed through the request body so that we can exit early without making any additional requests.